Add AI tool policy to contributing guide (#15002)
tauri-apps/tauri: .github/CONTRIBUTING.md

The JS API provides bindings between the developer's JS in the Webview and the built-in Tauri APIs, written in Rust. Its code is located in [/packages/api](https://github.com/tauri-apps/tauri/tree/dev/packages/api).
After making changes to the code, run `pnpm build` to build it. To test your changes, we recommend using the API example app, located in [/examples/api](https://github.com/tauri-apps/tauri/tree/dev/examples/api). It will automatically use your local copy of the JS API and provides a helpful UI to test the various commands.

## AI Tool Policy

It takes a lot of time to review a Pull Request while too easy to make a nonsense but plausible looking one using AI tools.

> It is unfair for other contributors and the reviewers to spend much of the time dealing with this, hence these rules:

Reviewer: ```suggestion
It takes a lot of time to review a Pull Request while it's very easy to make a nonsense but plausible looking one using AI tools.
It is unfair for other contributors and the reviewers to spend much of the time dealing with this, hence these rules:
```
perhaps nonsense should be nonsensical idk
also, the "spend much of the time dealing" part feels a bit weird but i can't come up with something

> 1. Review and test all LLM-generated content before submitting, you're the one responsible for it, not the AI.

Reviewer: ```suggestion
1. Review and test all LLM-generated content before submitting a PR, you're the one responsible for it, not the AI.
```

Reviewer: very unsure about this, your current version is fine as well since mine changes the meaning of the following `it` which i don't like

Author: I was being intentional vague here, submitting could be for issue, pr, ...

Reviewer: right, makes sense

> 2. Don't use AI to respond to review comments (execpt only for translations)

Reviewer: ```suggestion
2. Don't use AI to respond to review comments (except for translations).
```

> We will close the Pull Request with a `ai-slop` tag if you failed to do so.

Reviewer: silently closing it sounds funnier 😂

Author: 😂 I feel like it's more of a thing we could we to filter when looking back to closed PRs if anyone ever do (I certainly tried when looking at how other projects deal with this)
